File: source_files/ProjectB/Calculator/app/src/main/java/com/example/calculator/Calculator.java
package com.example.calculator;

public class Calculator {
    public static double add(double num1,double num2){
        return num1 + num2;
    }

    public static double subtract(double num1,double num2){
        return num1 - num2;
    }

    public static double multiply(double num1,double num2){
        return num1 * num2;
    }

    public static double divide(double num1,double num2){
        if(num2 == 0){
            throw new IllegalArgumentException("Cannot divide by 0");
        }
        return num1/num2;
    }
}

File: app/calculator.php
<?php
namespace App;
class calculator{
  

  //Defining functions so that they may be tested separately
  function add($one, $two){
    return $one + $two;
  }

  function subtract($one, $two){
    return $one - $two;
  }

  function multiply($one, $two){
    return $one * $two;
  }

  function divide($one, $two){
    return $one / $two;
  }
}


